webapp/src/reducers/index.js

```diff
@@ -6,6 +6,13 @@ import {combineReducers} from 'redux';
 import ActionTypes from '../action_types';
 import Constants from '../constants';
 
+const defaultSidebarContent = {
+    reviews: [],
+    prs: [],
+    assignments: [],
+    unreads: [],
+};
```

Comment:
I think it makes sense to have this defined right above `sidebarContent` instead of at the top of the file